src/Nox.Solution/Enums/Application/Integration/IntegrationSourceAdapterType.cs

```diff
@@ -0,0 +1,10 @@
+namespace Nox;
+
+public enum IntegrationSourceAdapterType
```

Comment:
I would suggest just to add the adapter that we are supporting at the moment
and comment the other ones
to avoid creating wrong expectations
and to be clear what we are supporting at the moment

for now is DatabaseQuery only?